framer/Layer.coffee

```diff
@@ -178,6 +178,7 @@ class exports.Layer extends BaseClass
 
 		# Sanitize calculated property setters so direct properties always win
 		layerPropertyIgnore(options, "point", ["x", "y"])
+		layerPropertyIgnore(options, "midPoint", ["midX", "midY"])
```

Comment:
Where is this used?

Reply:
Nowhere, it's public api to be able to do
`layer.midPoint = path.start`